SPECS/harfbuzz/CVE-2023-25193.patch

```diff
@@ -0,0 +1,513 @@
+diff -ruN a/src/hb-buffer.h b/src/hb-buffer.h
+--- a/src/hb-buffer.h	2022-02-12 14:31:41.000000000 -0800
++++ b/src/hb-buffer.h	2023-02-24 02:13:03.922391957 -0800
+@@ -377,7 +377,8 @@
+   HB_BUFFER_FLAG_PRESERVE_DEFAULT_IGNORABLES	= 0x00000004u,
+   HB_BUFFER_FLAG_REMOVE_DEFAULT_IGNORABLES	= 0x00000008u,
+   HB_BUFFER_FLAG_DO_NOT_INSERT_DOTTED_CIRCLE	= 0x00000010u,
+-  HB_BUFFER_FLAG_VERIFY				= 0x00000020u
++  HB_BUFFER_FLAG_VERIFY				= 0x00000020u,
++  HB_BUFFER_FLAG_PRODUCE_UNSAFE_TO_CONCAT	= 0x00000040u
```

Comment:
I don't see some of the changes (e.g. this unsafe_to_concat field) in either of the linked patches. Just curious, where did you grab the changes from?

Reply:
I grab this from [here](https://github.com/harfbuzz/harfbuzz/blob/07f2d8d5384943445ca00c0e127de81d37539e65/src/hb-buffer.h). They have updated those values in a previous [PR](https://github.com/harfbuzz/harfbuzz/commit/ab143e85c377512365134c1904b8f9f668309438).